source/Connection/AMDSTCPDataServer.cpp

```diff
@@ -153,7 +153,7 @@ void AMDSTCPDataServer::onClientRequestProcessed(AMDSClientRequest *processedReq
 
 	AMDSClientDataRequest *processedClientDataRequest = qobject_cast<AMDSClientDataRequest*>(processedRequest);
 	if(processedClientDataRequest && processedClientDataRequest->validateResponse()) {
-//		processedClientDataRequest->printData();
+		processedClientDataRequest->printData();
```

Comment:
This can be uncommented since we added the overridden debugMessage() function, which will only call `.toString()` for ClientRequestMessage when we really want to print the data